.github/hacktoberfest.md

```diff
@@ -22,4 +22,5 @@ These awesome people reviewed i18n PRs for us. Thank you! 💜
 
 - @agustinmulet reviewed #1609
 - @moustaphadev reviewed #1668
+- @moustaphadev reviewed #1654
```

Comment:
Not sure if I need to add my name again 🤔

Reply:
Why not! Let's get your name ALL OVER this document!!  🥳